mne/transforms.py

```diff
@@ -1759,6 +1760,10 @@ def apply_volume_registration(moving, static, reg_affine, sdr_morph=None,
     interpolation : str
         Interpolation to be used during the interpolation.
         Can be "linear" (default) or "nearest".
+    cval : float | str
+        The constant value to assume exists outside the bounds of the
+        ``moving`` image domain. Can be a string percentage like ``'1%%'``
```

Comment:
Is the `%` escaped so that it reads `1%`? Since Circle failed, I can't check, this is more of a note to self to check

Reply:
Yes because of the `verbose` / `fill_doc`